src/pages/Sketch/components/Canvas.module.css

```diff
@@ -49,9 +49,9 @@
 
 /* ----- Mouse Position ----- */
 .mousePosition {
-  position: absolute;
-  bottom: 11px;
-  right: 30px;
+  position: fixed;
```

Comment:
Convert this to scss please

Reply:
@smaranjitghose As you asked me before to open issues to convert css to scss... I will open those just right after my current assigned issues get completed. I am just facing a little trouble with some issues thats why its taking time

I will complete all my issue within one day

I will open css to scss issues just right after I finish my assigned work

Reply:
Sure!